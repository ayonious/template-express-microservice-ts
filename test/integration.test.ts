import request from 'supertest';
import app from '../src/controllers/testingEndpoints';

describe('Test with snapshot', () => {
  it(`test1`, async () => {
    const response = await request(app).get('/sum?val=2');
    expect(response.status).toBe(200);
    expect(response.text).toMatchSnapshot();
  });
});
