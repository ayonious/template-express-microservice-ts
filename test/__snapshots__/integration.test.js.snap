// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test with snapshot test1 1`] = `"sum of [1 to 2] is 3"`;
